### PR TITLE
Update strings.po

### PR DESCRIPTION
**Pull request by aviloria**
Update spanish strings.

### CODE REVIEW

File: language/Spanish/strings.po [hunk 1]
```diff
@@ -347,55 +348,55 @@ msgstr "Avance rápido"
...
 
 #: src/frontend/mame/ui/info.cpp:40
 msgid "LAN"
-msgstr ""
+msgstr "Red local"
 
 #: src/frontend/mame/ui/info.cpp:41
 msgid "WAN"
-msgstr ""
+msgstr "Red global/Internet"
```
Comment (cuavas) [inline language/Spanish/strings.po]: Does capitalised "Internet" imply connection to public Internet?  If so it shouldn't be here.  WAN adapters (ISDN, modem, etc.) are just as likely to connect to a proprietary network.  LAN adapters (e.g. Ethernet) may also be used to connect to public Internet.

File: language/Spanish/strings.po [hunk 2]
```diff
@@ -1095,15 +1106,18 @@ msgid ""
...
 "\n"
 msgstr ""
+"Falta la máquina seleccionada o una o más de sus ROMs o imágenes CHD. "
+"Por favor, elige otra máquina.\n"
+"\n"
 
 #: src/frontend/mame/ui/selgame.cpp:1737 src/frontend/mame/ui/selsoft.cpp:630
 msgid "Press any key to continue."
-msgstr ""
+msgstr "Pulsa cualquier tecla (menos ESC) para continuar."
```
Comment (cuavas) [inline language/Spanish/strings.po]: Adding "menos ESC" to this string is misleading.  Firstly, UI Cancel can be remapped, so assuming it's mapped to the escape key is not acceptable.  Secondly, this string is used in the missing software popup which can be dismissed with the UI Cancel key (escape by default), so it isn't even correct with default mappings.

File: language/Spanish/strings.po [hunk 3]
```diff
@@ -1538,22 +1552,22 @@ msgstr "Posición vertical %1$s"
 #: src/frontend/mame/ui/ui.cpp:1467
 #, c-format
 msgid "Laserdisc '%1$s' Horiz Stretch"
-msgstr "Estiramiento horizontal de Laserdisc «%1$s»"
+msgstr "Estiramiento horizontal de laserdisc «%1$s»"
```
Comment (cuavas) [inline language/Spanish/strings.po]: Are trademarks not capitalised in Spanish?  Laserdisc is a trademark, and hence treated as a proper noun and capitalised in Germanic languages.  (Honest question, I don't know how it's handled in Spanish.)

File: language/Spanish/strings.po [hunk 4]
```diff
@@ -2202,6 +2219,10 @@ msgid ""
 "\n"
 "Press any key to continue."
 msgstr ""
+"Al juego elegido le falta una o varias ROMs o imágenes CHD necesarias. "
+"Por favor elige otro distinto."
+"\n"
+"Pulsa cualquier tecla (menos ESC) para continuar."
```
Comment (cuavas) [inline language/Spanish/strings.po]: Please don't add "menos ESC" here either.

File: language/Spanish/strings.po [hunk 5]
```diff
@@ -2423,7 +2444,7 @@ msgstr "Elegir máquina nueva"
 #~ "Falta la máquina seleccionada o una o más de sus ROMs o imágenes CHD. Por "
 #~ "favor, elige otra máquina.\n"
 #~ "\n"
-#~ "Pulsa cualquier tecla para continuar."
+#~ "Pulsa cualquier tecla (menos ESC) para continuar."
```
Comment (cuavas) [inline language/Spanish/strings.po]: The strings in this section are from a previous version of MAME and no longer used (note that they're all preceded by `#` characters) - they're provided as reference to possibly help maintain terminology when translating updated strings.  There's no point modifying translations in this section.  You should remove this section after updating the translation.

File: language/Spanish/strings.po (hunk 1 above)
Comment (aviloria) [inline language/Spanish/strings.po]: Non-capitalized "internet" is a common name for Wide Area Networks (WANs), in fact it's more used than "global network" ("Red global")...
But, as you've said, it's commonly confused with "public Internet" connections.

File: language/Spanish/strings.po (hunk 2 above)
Comment (aviloria) [inline language/Spanish/strings.po]: Fixed! Thanks!

File: language/Spanish/strings.po (hunk 3 above)
Comment (aviloria) [inline language/Spanish/strings.po]: I was not sure about "Laserdisc" as a trademark or a technology... Reverted! Thanks!

File: language/Spanish/strings.po (hunk 4 above)
Comment (aviloria) [inline language/Spanish/strings.po]: Fixed!

File: language/Spanish/strings.po (hunk 5 above)
Comment (aviloria) [inline language/Spanish/strings.po]: Fixed!

**Pull request merged**